.rubocop.yml

```diff
@@ -17,5 +17,15 @@ Style/AlignParameters:
 Style/MultilineOperationIndentation:
   EnforcedStyle: indented
 
+Style/RedundantReturn:
+  Exclude:
+    - "chef/data_bags/crowbar/migrate/*"
+
 Lint/EndAlignment:
   AlignWith: variable
+
+Lint/UnusedMethodArgument:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 100
```

Comment:
really? ruby and python styleguides prefer < 80 (< 78 I think due to emailing diffs)